api_gateway/src/main/kotlin/org/thechance/api_gateway/data/model/image/ImageResponse.kt

```diff
@@ -0,0 +1,75 @@
+package org.thechance.api_gateway.data.model.image
+
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ImageResponse(
+    @SerialName("data")
+    val `data`: Data?,
+    @SerialName("success")
+    val success: Boolean?,
+    @SerialName("status")
+    val status: Int?
+){
+    @Serializable
+    data class Data(
```

Comment:
Does we need all these attributes ? if no , we can just still the attributes we use